multiqc/utils/search_patterns.yaml

```diff
@@ -30,7 +30,7 @@ fastqc:
     zip:
         fn: _fastqc.zip
 featurecounts:
-    fn: _counts.txt.summary
+    fn: counts.summary
```

Comment:
Instead of replacing the old version, maybe just keep both as an array instead?
```yaml
featurecounts:
    fn:
        - _counts.txt.summary
        - counts.summary
```
I think this should work, and keep backwards compatibility.